cmd/vic-machine/create/create.go

```diff
@@ -181,6 +182,12 @@ func (c *Create) Flags() []cli.Flag {
 		},
 
 		// volume
+		cli.StringFlag{
+			Name:        "default-volume-store, vsd",
+			Value:       "",
+			Usage:       "Specify the default volume store location, This location is used to support anonymous volumes such as image volumes. e.g.  \"datastore/path\" or \"datastore\".",
```

Comment:
Whichever wording you wind up with here, `", This"` is a comma splice so either make it two sentences or use a semicolon or avoid this wording entirely ;)